### PR TITLE
remove qt4 code

### LINKED ISSUE #5980: Remove a lot more Qt4 code
Noticed when reviewing @jturcotte 's https://github.com/owncloud/client/pull/5978 

there is still a ton of

```
src/cmd/cmd.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/accountsettings.cpp:#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
src/gui/activitylistmodel.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/gui/activitywidget.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/application.cpp:#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
src/gui/application.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/application.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0) && QT_VERSION < QT_VERSION_CHECK(5, 5, 0)
src/gui/folder.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/folderstatusdelegate.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/folderstatusdelegate.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/gui/folderstatusmodel.h:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/gui/folderwizard.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/issueswidget.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/main.cpp:#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
src/gui/owncloudgui.cpp:#if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)) && (QT_VERSION < QT_VERSION_CHECK(5, 6, 0))
src/gui/protocolwidget.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/selectivesyncdialog.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/settingsdialog.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/socketapi.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/socketapi.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/sslbutton.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/gui/sslbutton.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/gui/sslerrordialog.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/sslerrordialog.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/updater/ocupdater.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/gui/wizard/owncloudconnectionmethoddialog.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/gui/wizard/owncloudsetuppage.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/abstractnetworkjob.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/accessmanager.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0) && defined(Q_OS_MAC)
src/libsync/creds/httpcredentials.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 5, 0)
src/libsync/filesystem.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/filesystem.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/logger.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/logger.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/owncloudpropagator.h:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/propagatorjobs.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/syncengine.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/libsync/syncengine.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/libsync/syncengine.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/libsync/theme.cpp:#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
src/libsync/utility.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/utility.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/utility.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
src/libsync/utility.cpp:#elif QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/libsync/utility_unix.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
src/libsync/utility_unix.cpp:#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
test/testchecksumvalidator.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
test/testchecksumvalidator.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
test/testxmlparse.cpp:#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
```

### PR DESCRIPTION
**Pull request by tessus**
closes  #5980

### CODE REVIEW

File: src/common/utility.cpp [hunk 1]
```diff
@@ -410,19 +388,7 @@ QString Utility::platformName()
...
-        re = QLatin1String("MacOSX");
-    } else if (isLinux()) {
-        re = QLatin1String("Linux");
-    } else if (isBSD()) {
-        re = QLatin1String("BSD");
-    } else if (isUnix()) {
-        re = QLatin1String("Unix");
-    }
-#else
     re = QSysInfo::prettyProductName();
```
Comment (ogoffart) [inline src/common/utility.cpp]: could just be 
´´´
return QSysInfo::prettyProductName();
´´´

No need to initialize re to "Windows"

File: src/gui/accountsettings.cpp [hunk 2]
```diff
@@ -221,13 +221,7 @@ QString AccountSettings::selectedFolderAlias() const
 
 void AccountSettings::slotOpenAccountWizard()
 {
-    if (
-#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
-        qgetenv("QT_QPA_PLATFORMTHEME") == "appmenu-qt5" ||
-// We can't call isSystemTrayAvailable with appmenu-qt5 because it breaks the systemtray
```
Comment (ogoffart) [inline src/gui/accountsettings.cpp]: Please keep the comment!

File: src/gui/main.cpp [hunk 3]
```diff
@@ -132,11 +128,7 @@ int main(int argc, char **argv)
         }
         return 0;
     }
-#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
     if (qgetenv("QT_QPA_PLATFORMTHEME") != "appmenu-qt5")
-// We can't call isSystemTrayAvailable with appmenu-qt5 begause it hides the systemtray
```
Comment (ogoffart) [inline src/gui/main.cpp]: you must keep the comment.

**Comment by tessus** [comment]
@ogoffart how to you want me to do the change? 

- additional commit
- git reset; git push -f

Sorry for the dumb question, but I'm not familiar with your processes.

In any case, I'll do the change tomorrow. Have to get some shut-eye. Gotta get up in a few hours.

**Comment by ogoffart** [comment]
You can change your commit using `git commit --amend`  then you can push -f to your branch and that should update this pull request.

**Comment by tessus** [comment]
@ogoffart :-) I'm familiar with git. I just didn't know, if you wanted an additional commit for changes or only one commit in total (which requires an amendment or the use of git reset, for which you have to force push in both cases).

**Comment by tessus** [comment]
@ogoffart Done!

**Pull request merged**